packages/pyright-internal/src/tests/harness/fourslash/testState.ts

```diff
@@ -49,6 +50,9 @@ import {
     setTestingMode,
 } from '../../../common/pathUtils';
 import { convertOffsetToPosition, convertPositionToOffset } from '../../../common/positionUtils';
+import { ServiceProvider } from '../../../common/serviceProvider';
```

Comment:
why ordering changed? is pyright and pylance uses different prettier setting?

Reply:
The `.prettierrc` in pyrx and pyright are identical. The new ordering looks correct to me. Not sure why the ordering was wrong before this.